webpack/rules/javascript.js

```diff
@@ -1,18 +1,12 @@
 const PATHS = require('../paths');
 
 module.exports = ({ production = false, browser = false } = {}) => {
-  const enableHotModuleReplacement = !production && browser;
-  const createPresets = enableHotModuleReplacement => {
-    const presets = ['es2015', 'react', 'stage-0'];
-    return enableHotModuleReplacement ? ['react-hmre', ...presets]: presets;
-  };
-  const presets = createPresets(enableHotModuleReplacement);
-
+  const presets = ['es2015', 'react', 'stage-0'];
```

Comment:
should this be 

```javascript
const presets = [
    ['es2015', {modules: false}],
    'react',
    'stage-0'
]
```

to allow for `webpack` dead code elimination?

Reply:
@GGAlanSmithee Does that work in client environment too?

Reply:
@ZeroCho I don't see why it shouldn't. The webpack bundling (and therefor code elimiation) happends at "compile" time, so it shouldn't matter in what environment the final code is used. Is there any specific reason that makes you think it wouldn't work in browsers?

Reply:
@GGAlanSmithee You're right. I checked out that it works in client too